cmd/serverless/main.go

```diff
@@ -326,6 +326,11 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		// to detect the finished request spans and run the complete AppSec
 		// monitoring logic, and ultimately adding the AppSec events to them.
 		ta.ModifySpan = appsecProxyProcessor.WrapSpanModifier(serverlessDaemon.ExecutionContext, ta.ModifySpan)
+		// Set the default rate limiting to 1 trace/min to limit non ASM related traces as much as possible.
+		// In case of ASM event, the trace priority will be set to manual keep
+		if appsec.IsStandalone() {
+			ta.PrioritySampler.UpdateTargetTPS(1. / 120)
```

Comment:
I find it somewhat confusing that "1 per minute" is `1/120`? Can this be explained (at least to me, if that is clear to anyone else)?

Also - this looks like a magic number, which might be best shoved into a constant.

Reply:
It's an overall APM platform limitation based on constant streaming of traces and spans. Making it 0, which would be "send nothing but traces with appsec events", would break the billing pipeline and service catalog, which both expect receive traces. And we identified that the minimum we could get is 1 trace per minute.

Reply:
Worth further detailing this in the code comment indeed @Hellzy 🙏

Reply:
Enhanced the comment. As for the number, it indeed reflects 1/2min, but it's interestingly the threshold that reflects 1/min when continuously testing under various loads

Reply:
721cd746ac13e